### ISSUE TITLE
Surface 3 LTE internal MMC

### ISSUE THREAD

**Issue by morfeokmg** [jakeday/linux-surface#492]
Hello team, 

 I follow the instructions, but only in the step (ubuntu installation) when ubuntu was installed on my surface i lost my internal MMC and right now my surface is a expensive rock. 

My selection of installation
![photo_2019-05-06_20-08-56](https://user-images.githubusercontent.com/6172748/57265079-69af7a80-703b-11e9-853a-231844e0a014.jpg)

After end the installation
![photo_2019-05-06_11-05-18](https://user-images.githubusercontent.com/6172748/57265087-78962d00-703b-11e9-82e2-98d0b6f19e42.jpg)

the error:
![photo_2019-05-06_11-06-22](https://user-images.githubusercontent.com/6172748/57265094-821f9500-703b-11e9-8fc3-82205eec043e.jpg)

I receive the message (after the installation) 
MMC0: card has unknow MMCA version 6
MMC0: error -22 whilst initialising MMC card

I lost totally my surface because i can't reinstall windows because doesn't exist one internal disk to install windows. 
  Someone have the same problem?, do you know how can recovery the internal disk?
thanks in advance!
BR

**Comment by morfeokmg** [comment]
Surface Characteristics and SO: 
  Surface 3 (no pro) LTE 64 Gb internal
  Ubuntu Desktop 19.04

**Comment by StefanTT** [comment]
Did you try if it helps to disable LVM and encryption?

**Comment by morfeokmg** [comment]
No, is possible one bug in the Ubuntu 19.4 installer. Due i lost totally my internal MMC. 
right now, i installed Fedora 30 in one external SD, and it works, but with one USB bootloader.
